app/js/containers/HomeContainer.scss

```diff
@@ -8,6 +8,6 @@
     right: 0;
     bottom: 0;
     left: 0;
-    height: 7rem;
+    padding: 20px;
```

Comment:
I tend to prefer to use `rem` over `px` for sizing
What do you think?
Not valuable in this case as it is mobile only?

Reply:
I used the values from Figma and they are only in `px`. I agree that `rem` are better though.